### PR TITLE
#158853569 Create asset specs model

### PR DESCRIPTION
**Pull request by JaphethCA**
## What does this PR do?
- Creates AssetSpecs model 

## Description of tasks to be completed
- write tests for model
- adds AssetSpecs to admin panel

## Relevant Pivotal Tracker stories
[#158853569](https://www.pivotaltracker.com/story/show/158853569)

## Screenshot
screen shot 2018-07-09 at 2 57 13 pm

### CODE REVIEW

File: core/models/asset.py [hunk 1]
```diff
@@ -173,6 +240,10 @@ class Asset(models.Model):
     asset_condition = models.CharField(editable=False,
                                        max_length=50,
                                        default='Brand New')
+    specs = models.ForeignKey(AssetSpecs,
+                              blank=True,
+                              null=True,
+                              on_delete=models.CASCADE)
```
Comment (egichuri) [inline core/models/asset.py]: let's use `PROTECT` here..

Comment (egichuri) [reply core/models/asset.py]: while we're here.. please change https://github.com/AndelaOSP/art-backend/pull/123/files#diff-bea1207c116ad66b7b398915447bd091R129 to `PROTECT` as well

Comment (JaphethCA) [reply core/models/asset.py]: Done! Thanks for the feedback.

File: core/models/asset.py [hunk 2]
```diff
@@ -399,3 +470,27 @@ def allocation_history_post_save(sender, **kwargs):
             current_status=ALLOCATED
         )
         asset_status.save()
+
+
+@receiver(pre_save, sender=AssetSpecs)
```
Comment (egichuri) [inline core/models/asset.py]: do we need this validation when using the `choices` parameter?

**Comment by egichuri** [comment]
this is a test (ignore)

**Pull request merged**